qemu/tests/pvpanic.py

```diff
@@ -62,13 +64,28 @@ def trigger_crash(test, vm, params):
     :param vm: target vm
     :parma params: test params
     """
-    # to do: will add other crash method
-    crash_method = params.get("crash_method")
+    crash_method = params["crash_method"]
     if crash_method == "nmi":
         vm.monitor.nmi()
+    elif crash_method in ("usb_keyboard", "ps2_keyboard"):
+        crash_key1 = params["crash_key1"]
+        crash_key2 = params["crash_key2"]
+        vm.monitor.press_release_key(crash_key1)
+        vm.send_key(crash_key2)
+        vm.send_key(crash_key2)
+    elif crash_method == "notmyfault_app":
+        timeout = int(params.get("timeout", 360))
+        session = vm.wait_for_login(timeout=timeout)
+        cmd = params["notmyfault_cmd"] % random.randint(1, 8)
+        notmyfault_cmd = utils_misc.set_winutils_letter(session, cmd)
+        status, output = session.cmd_status_output(cmd=notmyfault_cmd,
+                                                   timeout=timeout)
```

Comment:
Will this operation crash the guest os? If yes, suggest to ignore some of the exception raised by this command.